README.md

```diff
@@ -103,12 +105,18 @@ it will install the required gemfile for running the project
 
 ## 👥 Author <a name="author"></a>
 
-   👤 **addisalem**
+  👤 **addisalem**
 
 - GitHub: [@addisalemseifu](https://github.com/githubhandle)
 - Twitter: [@Addisalem Seifu](https://twitter.com/addisalems92518)
 - LinkedIn: [addisalem seifu](https://www.linkedin.com/in/addisalemseifu/)
 
+👤 **Zohra Neda**
+
+- GitHub: [@zohra-neda](https://github.com/zohra-neda)
+- Twitter: [@zohra_neda](https://twitter.com/zohra_neda)
```

Comment:
- [ ] Please kindly ensure that all the social media handles are accessible and redirected to the expected pages. At the moment the github account for @addisalemseifu is a wrong one I guess, the twitter accounts for @Zohra-Neda does not exist; and that of @addisalemseifu is suspended. Kindly fix this in order to make your README file professional. **Hint: If the twitter accounts are not available at the moment, I think you can remove them** 🙏

Reply:
My partner dod not have a working twitter account for now. so I just removed it.